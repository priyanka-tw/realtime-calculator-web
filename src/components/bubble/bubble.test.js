import React from 'react';
import {render} from '@testing-library/react';
import 'jest-styled-components';
import Bubble from "../bubble/bubble";

describe('Bubble', () => {

    it('should render the component with given data', () => {
        const {container} = render(<Bubble data={"some random data"}/>);

        expect(container).toMatchSnapshot();
    });
});
