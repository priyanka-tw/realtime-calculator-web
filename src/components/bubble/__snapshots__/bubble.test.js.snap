// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bubble should render the component with given data 1`] = `
.c0 {
  width: 200px;
  margin: 10px auto;
  border: 3px solid black;
  padding: 10px;
  text-align: center;
  font-size: 14pt;
  color: black;
  font-family: verdana;
  border-radius: 25px;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
}

<div>
  <div
    class="c0"
  >
    some random data
  </div>
</div>
`;
